.github/workflows/release.yaml

```diff
@@ -0,0 +1,177 @@
+        ref: 986346a705799f528d620c19ca384e21cb008b54
+        path: SoulsIds
+    - uses: actions/checkout@v4
+      with:
+        repository: thefifthmatt/SoulsFormats
+        ref: dsms
+        path: SoulsFormats
+    - uses: microsoft/setup-msbuild@v2
+
+    - name: Install dependencies
+      run: dotnet restore SoulsRandomizers\SoulsRandomizers.sln
+
+    - name: Build randomizer
+      run: msbuild SoulsRandomizers\DS3Randomizer\DS3Randomizer.csproj '-property:Configuration=Release (Archipelago);PublishDir=bin\publish;PublishProtocol=FileSystem;_TargetId=Folder' '-t:Build;Publish'
+
+    - uses: actions/upload-artifact@v4
+      with:
+        name: DS3Randomizer.exe
+        path: SoulsRandomizers\DS3Randomizer\bin\publish\DS3Randomizer.exe
+        if-no-files-found: error
+
+  build-yaml:
+    runs-on: windows-latest
+    needs: [version]
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: nex3/Archipelago
+        ref: ds3-${{ needs.version.outputs.version }}
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: 3.11
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest pytest-subtests pytest-xdist
+        python ModuleUpdate.py --yes --force --append "WebHostLib\requirements.txt"
+
+    - name: Generate YAML
+      run: |
+        python Launcher.py 'Generate Template Options'
+        Move-Item -Path 'Players\Templates\Dark Souls III.yaml' -Destination 'Dark Souls III Options Template.yaml'
+
+    - uses: actions/upload-artifact@v4
+      with:
+        name: Dark Souls III Options Template.yaml
+        path: Dark Souls III Options Template.yaml
+        if-no-files-found: error
+
+  build-zip:
+    runs-on: windows-latest
+    needs: [version, build-mod, build-static, build-yaml]
+    env:
+      dir: DS3 Archipelago ${{ needs.version.outputs.version }}
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        path: Dark-Souls-III-Archipelago-client
+
+    - uses: actions/checkout@v4
+      with:
+        repository: nex3/SoulsRandomizers
+        ref: archipelago-server
+        path: SoulsRandomizers
+
+    - name: Download static files
+      run: |
+        Invoke-WebRequest -OutFile old.zip -Uri 'https://github.com/nex3/Dark-Souls-III-Archipelago-client/releases/download/v3.0.0-beta.1/DS3.Archipelago.2.3.0-beta.1.zip'
+        Expand-Archive old.zip -DestinationPath .
+
+    - name: Download ModEngine2
+      run: |
+        Invoke-WebRequest -OutFile ME2.zip -Uri 'https://github.com/soulsmods/ModEngine2/releases/download/release-2.1.0/ModEngine-2.1.0.0-win64.zip'
+        Expand-Archive ME2.zip -DestinationPath .
+
+    - name: Download Archipelago repo
+      run: |
+        Invoke-WebRequest -OutFile Archipelago.zip -Uri "https://github.com/nex3/Archipelago/archive/refs/tags/ds3-${{ needs.version.outputs.version }}.zip"
+        Expand-Archive Archipelago.zip .
```

Comment:
Can we remove this from the final version once the server and the client has been merged ?